docs/rgb.md

```diff
@@ -0,0 +1,105 @@
+# RGB/Underglow/Neopixel
+Want your keyboard to shine? Add some lights!
+This does require the neopixel library from Adafruit. This can be downloaded here https://github.com/adafruit/Adafruit_CircuitPython_NeoPixel/blob/master/neopixel.py
```

Comment:
I believe Adafruit prefers linking to [the CircuitPython Bundle pack](https://github.com/adafruit/Adafruit_CircuitPython_Bundle) for stuff like this? That provides _way_ more than just NeoPixel of course.

@tannewt do you have a preference for this since inevitably if something breaks y'all will probably end up hearing about it? :joy:

Reply:
I will note that it's just a helper library. I almost completely stripped it as it's REALLY not needed but I wanted to allow users to interact with something they were familiar with directly and keep the CircuitPython spirit of availability of tools.